### ISSUE TITLE
ERROR: Can't process email, you will find it for bug reporting under blah.eml, please create an issue at https://github.com/zammad/zammad/issues

### ISSUE THREAD

**Issue by chesty** [zammad/zammad#2052]
### Infos:

* Used Zammad version: 2.4.x
* Installation method (source, package, ..): zammad-docker-compose

I emailed the eml, [Ticket#1020590]
This is just a FYI report because the log asked for a ticket. Please do of it as you like.

cheers, love your work.

**Comment by thorsteneckel** [comment]
Hi @chesty - I wasn't able to reproduce it on latest develop/upcoming 2.5. Please consider updating and let us know if the issue still persists. Thanks for reporting!

**Comment by chesty** [comment]
Thanks thorsteneckel,

would the subject size setting matter? mine is the default 110 and the subject looks longer.
I'll do some testing, I tried quanting how to resend the eml but didn't see the answer straight away, maybe you could save me 5 minutes, how do you send the eml?
I think I should be able to netcat it, there's a non ssl port open, I guess I need to prepend an envelope.

**Comment by thorsteneckel** [comment]
Yes, this might actually be the case. The subject is 118 chars long. There is an easy way to import it right away via the CLI: `cat /path/to/c6d6f462db3013cf8fbab678d2c36785.eml | rails r 'Channel::Driver::MailStdin.new'`

**Comment by chesty** [comment]
frustrating, it imports clean now with both the cat and IMAP even with subject size 100. I'll let you know if I see any more unprocessable mail. cheers.

**Comment by chesty** [comment]
Today I've had 1 email so far turn up in the unprocessable_mail folder and it went in OK when I catted it through `rails r 'Channel::Driver::MailStdin.new'`
Yesterday there were 2 emails the same.
We'll be updating to 2.5 when the docker-compose-zammad is updated

**Comment by martini** [comment]
@chesty just a note here, since Zammad 2.4 you can use `rails r 'Channel::EmailParser.process_unprocessable_mails'` to re-process all tickets from unprocessable directory. It makes it easier.

**Comment by chesty** [comment]
It seems to me things scheduled often fail, like 5 times a day (out of 1000s or more of scheduled jobs) like imap and ldap sync. If I rerun them from rails console I can't get them to fail.
